app/brave_generated_resources.grd

```diff
@@ -807,7 +807,7 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
         Method to resolve Unstoppable Domains
       </message>
       <message name="IDS_SETTINGS_RESOLVE_UNSTOPPABLE_DOMAINS_SUB_DESC" desc="The sub-description for how to handle Unstoppable Domains">
-        <ph name="BEGIN_LINK">&lt;a target="_blank" rel="noopener noreferrer" href="$1"&gt;</ph>Learn More<ph name="END_LINK">&lt;/a&gt;</ph> about DNS over HTTPS and Ethereum privacy considerations.
+        <ph name="BEGIN_LINK">&lt;a target="_blank" href="$1"&gt;</ph>Learn More<ph name="END_LINK">&lt;/a&gt;</ph> about DNS over HTTPS and Ethereum privacy considerations.
```

Comment:
this is generally fine since noopener is now the default behavior: https://www.chromestatus.com/feature/6140064063029248. noreferrer is not super important here.